noggin/tests/unit/controller/test_group.py

```diff
@@ -30,6 +30,24 @@ def test_groups_list(client, logged_in_dummy_user, dummy_group):
     assert group_desc.get_text(strip=True) == "A dummy group"
 
 
+@pytest.mark.vcr()
+def test_groups_list_no_hidden(client, logged_in_dummy_user, dummy_group):
+    """Test the groups list: /groups/"""
+    result = client.get('/groups/')
+    assert result.status_code == 200
+    page = BeautifulSoup(result.data, 'html.parser')
+    groups = page.select("ul.list-group li")
+    group_names = [g.find("span", class_="title").get_text(strip=True) for g in groups]
+    assert "ipausers" not in group_names
```

Comment:
Ideally our testing system would ensure that the hidden group exists on the server and put the `ipausers` group in there. A session-wide fixture could to the trick, so it's only done once per run.

Reply:
OK, I've added that fixture and pushing now. Am I right in saying that this test won't change though and you just mean that our test can reliably check that the group exists? Or am I missing something?

Reply:
Yes, that's the idea.